Formula/lepton.rb

```diff
@@ -0,0 +1,23 @@
+class Lepton < Formula
+  desc "Tool and file format for losslessly compressing JPEGs"
+  homepage "https://github.com/dropbox/lepton"
+  url "https://github.com/dropbox/lepton/archive/1.0.tar.gz"
+  sha256 "2ec19eee0f4e78de4cb6a1d48c2cb2615de77abeb1bb22c17d8cf90cd725046a"
+
+  depends_on "cmake" => :build
+
+  def install
+    # Wrong header, fixed in https://github.com/dropbox/lepton/commit/d122450
+    inreplace "src/vp8/util/memory.cc", "<emmintrin.h>", "<immintrin.h>"
```

Comment:
Patches are a bit more self-documenting and tend to get removed more readily when not needed so if you could remove it that'd be great.